docs/querying/sql.md

```diff
@@ -413,6 +400,29 @@ SET timeout = 90000;
 SELECT some_column, COUNT(*) FROM druid.foo WHERE other_column = 'foo' GROUP BY 1 ORDER BY 2 DESC
 ```
 
+If present, these `SET` statements assign [SQL query context parameter](../querying/sql-query-context.md) which only apply to the non-`SET` statement of the same request and subsequent requests are not affected.
+
+You can mix `SET` statement and [Context Parameter](../querying/query-context.md) with `SET` overriding on key conflicts.
+
+### Current limitations
+
+- `SET` statements currently support literal values only. For example, numbers, strings, booleans.
+-  You must still provide arrays or JSON objects via the `context` field.
+
+
```

Comment:
This is really two ways to say the same thing. We don't need to create a new section for just one limitation.

```suggestion
Note that you can only use SET to assign literal values, such as numbers, strings, or Booleans. To set a query context parameter to an array or JSON object, use the `context` field rather than SET.
```